main/src/main/res/values/strings.xml

```diff
@@ -2783,6 +2783,7 @@
     <string name="maptheme_elevate_categoryinfo">Relevant for: [A] Areas, [P] POI, [R] Routes, [W] Ways</string>
     <string name="maptheme_scale_map_title">Map Scale Factor</string>
     <string name="maptheme_scale_map_summary">Scales map; 100% = no scaling</string>
+    <string name="maptheme_scale_map_summary_vtm">Scale map (OpenStreetMap offline maps only), 100% = no scaling (requires restart)</string>
```

Comment:
I think we can shorten it a bit

```suggestion
    <string name="maptheme_scale_map_summary_vtm">Scale offline maps (requires restart)</string>
```

Reply:
(different approach taken meanwhile, therefore marking as resolved here)